src/app/actions.ts

```diff
@@ -36,6 +36,7 @@ export enum Actions {
   INVITE_LINK_CONSUMED = 'APP/INVITE_LINK_CONSUMED',
   HAPTIC_FEEDBACK_SET = 'APP/HAPTIC_FEEDBACK_SET',
   PUSH_NOTIFICATIONS_PERMISSION_CHANGED = 'APP/PUSH_NOTIFICATIONS_PERMISSION_CHANGED',
+  SET_STATSIG_LOAD_TIME = 'APP/SET_STATSIG_LOAD_TIME',
```

Comment:
I know the codebase is not following this in a bunch of places, but it's better to model actions as events instead of setters.
https://redux.js.org/style-guide/#model-actions-as-events-not-setters

Maybe?
```suggestion
  STATSIG_LOADED = 'APP/STATSIG_LOADED',
```
And we can rename the rest after this.

Reply:
Oh I didn't know this. Thanks! I actually ended up removing this state per @kathaypacific's suggestion to refresh always and not manage this state. This was also buggy, it doesn't reflect the correct value if the statsig user is patched with another field. We might end up invoking statsig more often, but this shouldn't cost us. Statsig continues to use previous values while this is refreshing.